alerting/src/main/kotlin/org/opensearch/alerting/DocumentLevelMonitorRunner.kt

```diff
@@ -236,6 +236,28 @@ class DocumentLevelMonitorRunner : MonitorRunner() {
                         }
                     }
 
+                    val fieldsToBeQueried = mutableSetOf<String>()
+
+                    for (it in queries) {
+                        if (it.queryFieldNames.isEmpty()) {
```

Comment:
Will we get an NPE here for migrations from older versions that did not have a queryFieldNames field in the DocLevelQuery objects?

Reply:
The constructor will  initialize to empty list if not present in index.